source/documentation/ways-of-working/index.html.md.erb

```diff
@@ -6,3 +6,206 @@ review_in: 3 months
+| [Peer Review](#peer-review)                   | Fortnightly | Thursday        | 15:30 - 16:30 |
+| [Refinement 2](#refinement)                   | Fortnightly | Thursday        | 12:00 - 12:45 |
+| [Sprint Retrospective](#sprint-retrospective) | Fortnightly | Thursday        | 11:00 - 12:00 |
+| [Sprint Planning](#sprint-planning)           | Fortnightly | Thursday        | 13:00 - 14:00 |
+
+### Stand Up
+
+Our stand-ups run daily at 10:45 and last for fifteen minutes.
+We use Google Meet and follow one of two formats:
+
+- **Walk The Board**
+
+  - Starting with the “Done” column we discuss any active issues on our
+  [GitHub Project Board](https://github.com/orgs/ministryofjustice/projects/57/views/5).
+
+- **Yesterday, Today, Blocked**
+
+  - A round robin of each team member discussing their issues and any blockers.
+
+Standup is run by either the PM, DM, or whoever wants to run it.
+
+Ad-hoc discussions may spin out from stand-up.
+These should take place after the meeting ends if time allows,
+or should be arranged for the next available time.
+
+### Show and Tell
+
+- Near the beginning of each sprint we have a thirty-minute show and tell showcase that provides an opportunity to share our progress from the previous sprint with the wider organisation.
+- [TODO: show and tell content]
+
+#### Audience
+
+All of Data Platform and any wider audience
+
+### Roadmap Review
+
+- ... share with our colleagues and the wider organisation ...
+- ... consists of an ... what’s next in the road map
+
+#### Audience
+
+All of the Data Improvement Programme and any wider audience
+
+### Refinement
+
+**Purpose:**
+
+We use refinement to align the team's understanding and agree on the units of work for upcoming sprints.
+It's where we ensure everyone has an understanding of the backlog items before they are added to a sprint.
+
+**Preparation:**
+
+Here's what can be prepared prior to engaging in a refinement session:
+
+- **Backlog overview:**
+Review the backlog to get a sense of what's coming up, and what's already been done.
+Read the tickets likely to come up for discussion, so you know what questions you want to ask.
+- **Backlog item review:**
+Ensure any tickets are well fleshed out and ready for discussion.
+This should include a detailed description of the story, as well as acceptance criteria.
+- **Understand Dependencies:**
+If your tickets rely on other work already being done,
+know which tickets are required and list them in the body of the issue.
+This helps the team establish sequencing of the tickets during refinement.
+- **Prepare to Communicate!:**
+Be prepared to articulate why you think a ticket should be done, how big you think it is,
+be receptive to teammates, and collaborate to achieve a shared understanding of the tasks at hand.
+Be open to suggestions on how your ticket could be improved for someone picking it up.
+
+**Process:**
+
```

Comment:
I like these guidelines - will be helpful if Sean is away and somebody else needs to run this meeting. Might be useful to copy this over into the calendar invite as well (or link to this)